src/commands/functions/functions-create.js

```diff
@@ -565,6 +624,28 @@ const installAddons = async function (command, functionAddons, fnPath) {
   return Promise.all(arr)
 }
 
+const registerEFInToml = async (funcName) => {
+  if (!fs.existsSync('netlify.toml')) {
+    log(`${NETLIFYDEVLOG} \`netlify.toml\` file does not exist yet. Creating it...`)
+  }
+
+  // Todo: validate this input
+  const { funcPath } = await inquirer.prompt([
+    {
+      type: 'input',
+      name: 'funcPath',
+      message: `What route do you want your edge function to be invoked on? \nRout must begin with '/'\
```

Comment:
This is probably what you mean with the `Todo:` comment above, but rather than saying that the route must begin with `/`  we could be more forgiving with the input and add the `/` ourselves.

Reply:
Makes sense. I kept forgetting the `/` and having to fix it manually to test the functions so I assumed other people would forget it as well. I'll add a check to prepend the `/` if it's missing instead.

Reply:
Typo on `Rout`.